scripts/test/lint.py

```diff
@@ -86,6 +86,26 @@ def run_cmd(cmd):
     return out
 
 
+class IncludesDescriptionLength(Lint):
+    max_length = 175
+    min_length = 10
+    name = "the description length is invalid"
+    recommendation = "See the VM-Packages Wiki for conventions and package structure related to the description"
+
+    def check(self, path):
+        dom = minidom.parse(str(path))
+        metadata = dom.getElementsByTagName("metadata")[0]
+        description = metadata.getElementsByTagName("description")[0]
+        text = description.firstChild.data
```

Comment:
I think you can access the description directly without accessing the metadata first.

[nitpick] I think we do not need the description variable since it is used once and IMO does not increase readability.

```suggestion
        dom = minidom.parse(str(path))
        description = dom.getElementsByTagName("description")[0].firstChild.data
```